api/src/pcapi/routes/backoffice/__init__.py

```diff
@@ -42,6 +42,6 @@ def install_routes(app: Flask) -> None:
     from .venues import blueprint as venues_blueprint
 
     if settings.ENABLE_TEST_USER_GENERATION:
```

Comment:
Je suis personnellement pour n'avoir qu'un seul setting, mais si tu laisses les deux, il faut envisager le cas où seul le second est activé :
```suggestion
    if settings.ENABLE_TEST_USER_GENERATION or settings.ENABLE_USER_DELETION:
```